build/config/RepoUtilData.json

```diff
@@ -1,26 +1,9 @@
 ﻿{
   "fixedPackages": {
-    "Microsoft.VisualStudio.Language.Intellisense": "14.3.25407",
     "Microsoft.VSSDK.BuildTools": [ "14.3.25420", "15.0.25929-RC2" ],
     "Newtonsoft.Json": "9.0.1",
     "System.Reflection.Metadata": "1.0.21",
     "System.Collections.Immutable": "1.1.36",
```

Comment:
We can probably do it in a separate PR, but I believe last time I talked to @jmarolf about this, we agreed it was reasonable to move the SDK package forward to v2.0